osu.Game/Database/ArchiveModelManager.cs

```diff
@@ -589,7 +589,9 @@ public Task ImportFromStableAsync()
                 return Task.CompletedTask;
             }
 
-            return Task.Run(async () => await Import(GetStableImportPaths(GetStableStorage()).Select(f => stable.GetFullPath(f)).ToArray()));
+            var directory = stable.GetStorageForDirectory(ImportFromStablePath);
```

Comment:
It's really confusing that many other usages have started using `.` as the directory but this hasn't...?

Reply:
`ArchiveModelManager` now access only to a `Storage` to their `ImportFromStablePath` directory to import data from stable instead of a storage at the root of the stable installation
hence the usage of "." for enumerating directories in `GetStableImportPaths`